client/asset/btc/btc.go

```diff
@@ -109,6 +109,26 @@ var (
 		},
 	}
 
+	// 02 Jun 21 21:12 CDT
+	defaultWalletBirthdayUnix = 1622668320
+	defaultWalletBirthday     = time.Unix(int64(defaultWalletBirthdayUnix), 0)
+
+	spvOpts = []*asset.ConfigOption{{
+		Key:         "walletbirthday",
+		DisplayName: "Wallet Birthday",
+		Description: "This is the date the wallet start scanning the blockchain for " +
+			"transactions related to this wallet. A rescan of the will automatically" +
+			"be started if the new birthday is earlier than the old one. This option" +
```

Comment:
Missing trailing space on this line and the previous.

![image](https://user-images.githubusercontent.com/9373513/156242012-3450446f-3fed-4bbd-aa66-5e7bfb1fb02e.png)

Reply:
The part about starting a rescan if "earlier than the old one" is pretty confusing when viewed from the wallet **creation** dialog rather than settings/reconfig.  To make this sensible in both context, how about "If reconfiguring an existing wallet, this may start a rescan if the new birthday is older."